### PR TITLE
Fix some typos and grammar in /exporters.md

### PR DESCRIPTION
**Pull request by windsonsea**
When reading this page, find some typos and grammar issue.
```
content/en/docs/instrumentation/js/exporters.md
```

### CODE REVIEW

File: content/en/docs/instrumentation/js/exporters.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ accessible for everyone.
 
 It is recommended that you do not expose your collector directly, but that you
 put a reverse proxy (nginx, apache, ...) in front of it. The reverse proxy can
-take care of SSL-offloading, setting the right CORS headers and many other
+take care of SSL-offloading, and set the right CORS headers and many other
```
Comment (naseemkullah) [inline content/en/docs/instrumentation/js/exporters.md:143]:
```suggestion
take care of SSL-offloading, set the right CORS headers and many other
```

File: content/en/docs/instrumentation/js/exporters.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ weight: 5
 
 In order to visualize and analyze your traces, you will need to export them to a
 backend such as [Jaeger](https://www.jaegertracing.io/) or
-[Zipkin](https://zipkin.io/). OpenTelemetry JS provides exporters for some
+[Zipkin](https://zipkin.io/). OpenTelemetry JavaScript provides exporters for some
```
Comment (naseemkullah) [inline content/en/docs/instrumentation/js/exporters.md:8]: Not sure if this one is necessary, the `JS` abbreviation is well known

**Review by naseemkullah — APPROVED** [review]

File: content/en/docs/instrumentation/js/exporters.md (hunk 1 above)
Comment (naseemkullah) [inline content/en/docs/instrumentation/js/exporters.md:143]: I would revert this one, look at it like this:


The reverse proxy can take care of:
- SSL-offloading
- setting the right CORS headers
- and many other features...

**Review by naseemkullah — CHANGES_REQUESTED** [review]

File: content/en/docs/instrumentation/js/exporters.md (hunk 1 above)
Comment (cartermp) [reply content/en/docs/instrumentation/js/exporters.md:143]: Yes, agreed

**Comment by windsonsea** [comment]
Thanks for your careful review. It's helpful to my learning :)

Reverted and modified by respecting all comments. 👍

**Comment by svrnm** [comment]
thank you for your contribution @windsonsea !

**Review by svrnm — APPROVED** [review]
lgtm

**Pull request merged**